packages/@aws-cdk/aws-iam/test/test.instance-profile.ts

```diff
@@ -0,0 +1,292 @@
+import { expect, haveResource } from '@aws-cdk/assert';
+import { Construct } from '@aws-cdk/cdk';
+import { Stack } from '@aws-cdk/cdk';
+import assert = require('assert');
+import { Test } from 'nodeunit';
+import { InstanceProfile,
+    InstanceProfileRef,
+    PolicyStatement,
+    Role,
+    ServicePrincipal
+} from '../lib';
+
+/**
+ *  Test Cases:
+ *  parameters: role
```

Comment:
I don't think this is necessary, and will likely become stale very quickly

Reply:
this was my 'punch list', I will remove.